src/software/ai/intent/spinning_move_intent_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include "software/ai/intent/spinning_move_intent.h"
+
+#include <gtest/gtest.h>
+#include <string.h>
+
+TEST(SpinningMoveIntentTest, intent_name_test)
+{
+    SpinningMoveIntent spinning_move_intent =
+        SpinningMoveIntent(0, Point(), Angle::zero(), 1.0, 0);
+
+    EXPECT_EQ("SpinningMove Intent", spinning_move_intent.getIntentName());
```

Comment:
Should we have the space in the name now?

Reply:
fixed